### PR TITLE
Fix for CTS Media Perf Test Cases

### PR DESCRIPTION
**Pull request by stode**
Change perf range in XML for TGL

Tracked on: OAM-97895
Signed-off-by: Shreya Tode <shreya.tode@intel.com>
Change-Id: I36c6729117a25f33b12a92eecbb0ac35ff8243b5

### CODE REVIEW

**Comment by k-mochi** [comment]
@JeevakaPrabu please review.

File: groups/codecs/configurable/media_codecs_performance_tgl.xml [hunk 1]
```diff
@@ -31,16 +31,16 @@
             <Limit name="measured-frame-rate-640x360" range="200-220" />
             <Limit name="measured-frame-rate-720x480" range="170-190" />
             <Limit name="measured-frame-rate-1280x720" range="70-120" />
-            <Limit name="measured-frame-rate-1920x1080" range="40-70" />
+            <Limit name="measured-frame-rate-1920x1080" range="159-240" />
```
Comment (TianmiChen) [inline groups/codecs/configurable/media_codecs_performance_tgl.xml:34]: Why this performance data is much more than the data for 720P?

Comment (k-mochi) [reply groups/codecs/configurable/media_codecs_performance_tgl.xml:34]: we will check all perf TCs and update PR. Thanks

**Comment by k-mochi** [comment]
this PR is replaced by https://github.com/projectceladon/device-androidia-mixins/pull/1189. hence closing this PR.

**Pull request closed without merging**